modmail/__init__.py

```diff
@@ -38,14 +42,15 @@
 
 file_handler.setLevel(logging.TRACE)
 
-coloredlogs.install(
-    level=LOG_LEVEL,
-    fmt=fmt,
-    datefmt=datefmt,
-)
+# configure trace color
+_LEVEL_STYLES = dict(coloredlogs.DEFAULT_LEVEL_STYLES)
```

Comment:
Can we make this configurable as `env` variable, it really helps when someone is using a weird colour scheme.

Reply:
This hasn't been mentioned yet.